topics/connect-to-platform-specific-apis.md

```diff
@@ -135,6 +135,20 @@ internal actual class PlatformSocket actual constructor(url: String) {
 }
 ```
 
+Android implementation uses [OkHttp](https://square.github.io/okhttp/) third-party library, so you need to declare the corresponding dependency in the shared module `build.gradle`:
+
+```kotlin
+sourceSets {
+    val androidMain by getting {
+        dependencies {
+            implementation("com.squareup.okhttp3:okhttp:4.9.0")
```

Comment:
Is it OK to use specific version here? Will it work in the future?

Reply:
Fixed